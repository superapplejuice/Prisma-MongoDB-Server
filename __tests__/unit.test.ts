import userTests from './user.test'

describe('User Resolvers: Registration, Login and Logout', userTests)
