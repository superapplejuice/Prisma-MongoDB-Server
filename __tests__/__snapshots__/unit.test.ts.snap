// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`User Resolvers: Registration, Login and Logout A new user should be registered 1`] = `
Object {
  "cart": Array [],
  "email": "test@gmail.com",
  "id": "5eca2d88e03dd80008794953",
  "items": Array [],
  "joined": "2020-05-24T08:17:12.696Z",
  "password": "$2a$12$1aBg2eN1j.bhb3/Yc8JSVeMj3WVLhnItWi7G9yFauJBKuBRxbo69a",
  "username": "testuser",
}
`;
